src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -95,6 +97,10 @@ class MockClientStats {
   size_t num_async_stream_infer_calls{0};
   size_t num_start_stream_calls{0};
 
+  std::atomic<size_t> num_active_infer_calls{0};
+
+  std::chrono::milliseconds request_delay{0};
```

Comment:
Seems like this should be a variable in the mock client, not in the stats

Reply:
can we rename this to response_delay?

Reply:
Renamed 👍